features/utils/pageobjects/detail.tour.page.js

```diff
@@ -0,0 +1,43 @@
+import Input from '../elements/input';
+import Button from '../elements/button';
+
+class DetailTourPage {
+  get buttonMaximumAmountOfDays() {
+    return new Button(':nth-child(1) > :nth-child(6) > .btn ');
+  }
+
+  get inputFirstName() {
+    return new Input('[name = \'firstname\']');
+  }
+
+  get inputPhone() {
+    return new Input('[name =\'phone\']');
+  }
+
+  get inputEmail() {
+    return new Input('.form-group > [name =\'email\']');
```

Comment:
Consider passing the general rule for all strings passing to the selectors :
 get buttonForIncreaseAmountOfAdults() { 
return new Button('#tours input[name="adults"] ~ .input-group-btn-vertical > button[class*=
**"**bootstrap-touchspin-up**"** 
in this case and   
get inputEmail() {
    return new Input('.form-group > [name =**\'**email**\'**]');
  }
in other case